packages/governance/test/swingsetTests/committeeBinary/bootstrap.js

```diff
@@ -4,50 +4,77 @@ import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
 
+import {
+  ChoiceMethod,
+  QuorumRule,
+  ElectionType,
+  looksLikeQuestionSpec,
+} from '../../../src/question.js';
```

Comment:
It doesn't look like `ElectionType` and so on are exported from the `@agoric/governance` package itself; they have to be "deep imported" like this. Is that by design?